<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDBCommands</class>
 <widget class="QWidget" name="MDBCommands">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>84</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QPushButton" name="pb_step">
     <property name="text">
      <string>Step</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pb_continue">
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_evaluate">
     <property name="text">
      <string>Evaluate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pb_rbreak">
     <property name="text">
      <string>Add break</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pb_next">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pb_break">
     <property name="text">
      <string>Break</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pb_finish">
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_forwardtrace">
     <property name="text">
      <string>Forwardtrace</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pb_backtrace">
     <property name="text">
      <string>Backtrace</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pb_frame">
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pb_help">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="pb_quit">
     <property name="text">
      <string>Quit mdb</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
